docs/api/apiv3/components/schemas/non_working_day_model.yml

```
@@ -0,0 +1,40 @@
+# Schema: NonWorkingDayModel
+---
+type: object
+required:
+  - _type
+  - date
+  - name
+properties:
+  _type:
+    type: string
+    enum:
+      - NonWorkingDay
+  date:
+    type: string
+    format: date
+    description: Date of the non-working day.
+  name:
+    type: string
+    description: Descriptive name for the non-working day.
+  _links:
+    type: object
+    required:
+      - self
+    properties:
+      self:
+        allOf:
+          - $ref: './link.yml'
+          - description: |-
+              This non working day. If the non-working day is recurring like a weekend day, the resource will point to a
+              week day.
```

```suggestion
              This is a non working day. If the non-working day is recurring like a weekend day, the resource will point to a
              week day.
```

I'm not sure I understand the second sentence, what is the difference between a non-working day and a weekday?

---

It should be read "Links to this non-working day" rather than "This non-working day". It is the description of the link. And it refers to the fact that there are multiple reasons for a dayu to be a non-working one.

How would you rephrase it?

---

It has changed with 314190da32.